### PR TITLE
Add Developer's table of contents / introduction.

### PR DESCRIPTION
**Pull request by zkamvar**
Adding the rationale and skeleton for developers' tutorials.

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -1,8 +1,69 @@
...
-* [R packages available for population genetics](PACKAGES.html)
-* [Data formats](DATAFORMATS.html)
-* [How to write an R package](https://github.com/jtleek/rpackages)
-* [How to contribute your workflow vignette here](CONTRIBUTING.html)
+Introduction
+=======
+
+You are reading this introduction because you are interested in contributing an
+R package to the population genetics community. You might have some R scripts
+lying around that implement a specific calculation, or possibly implements a new
```
Comment (hlapp) [inline DEVELOPER.md]: Remove _implements_.

File: DEVELOPER.md [hunk 2]
```diff
@@ -1,8 +1,69 @@
...
+    - Submitting to CRAN
+    
+A word on modularity
+=======
+
+R is an open source language with a strong community of developers contributing
+new methods and analyses. The population genetics community has worked hard to 
+create data classes that can be used efficiently between packages, expanding the
+number of analyses that can be done by users in a reproducible manner. Utilizing
+common data types within your R package lowers the barrier between users current
```
Comment (hlapp) [inline DEVELOPER.md]: _users'_ (apostrophy), and presumably _workflow_ should be plural?

**Comment by hlapp** [comment]
Looks great!

File: DEVELOPER.md (hunk 2 above)
Comment (zkamvar) [inline DEVELOPER.md]: Good point. That is a bit awkward, isn't it? How about: 

> ~~users~~ current workflows and your new method or visualization.

Comment (hlapp) [inline DEVELOPER.md]: :+1:

**Pull request merged**